src/UnitTests/ShouldMapMethod.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Xunit;
+using Shouldly;
+using System;
+
+namespace AutoMapper.UnitTests
+{
+    public class ShouldMapMethod : NonValidatingSpecBase
+    {
```

Comment:
This looks good, but the extension method case needs a test too.

Reply:
Ok. I'm gonna add it.